Datadog.Trace.sln

```diff
@@ -430,6 +430,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DuplicateTypeProxy", "test\
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Samples.AspNetCoreRazorPages", "test\test-applications\integrations\Samples.AspNetCoreRazorPages\Samples.AspNetCoreRazorPages.csproj", "{1B9E6BF4-9D48-4988-9945-248096119E46}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Samples.Kafka", "test\test-applications\integrations\Samples.Kafka\Samples.Kafka.csproj", "{760B1586-23B4-418A-B7D9-ABC0AC023FAE}"
```

Comment:
I don't know how the `FAE04EC0` keep sneaking in, but I'm pretty sure this should be `9A19103F`. If you remove the project and re-add it with Visual Studio, it will change the GUID. (The GUIDs for `Samples.AspNetCoreRazorPages` above and `FeatureTracking` below are already getting fixed in another PR: #1463)

Reply:
I wonder if it's due to Rider. I created the razor pages one and this one. Tony created Feature Tracking (I'm guessing with Rider). 

If only _.sln_ files were actually understandable. Doesn't seem to make any difference which GUID I use!

Reply:
Also, with 100+ projects...

![image](https://user-images.githubusercontent.com/18755388/118155836-8f34a300-b410-11eb-8483-4a39e5f65b78.png)

Reply:
Added and removed in https://github.com/DataDog/dd-trace-dotnet/pull/1444/commits/a9f0a2c6d9c7b5d7d536031f6bf87dd93c1f6d98... VS did...stuff...